test/srt/test_swa_unittest.py

```diff
@@ -40,10 +40,8 @@ def test_swa_memory_pool(self):
             full_attention_layer_ids,
             device,
         )
-        alloc = SWATokenToKVPoolAllocator(size, size_swa, dtype, device, pool)
-        assert alloc.available_size() == size + size_swa
+        alloc = SWATokenToKVPoolAllocator(size, size_swa, dtype, device, pool, True)
```

Comment:
Can you explain why delete the line `assert alloc.available_size() == size + size_swa`, thanks.

Reply:
In #7367 , it refactors SWATokenToKVPoolAllocator and removed available_size() to "NotImplemented". The UT is not updated accordingly.

```
class SWATokenToKVPoolAllocator(BaseTokenToKVPoolAllocator):
    """Allocator for SWA hybrid KV cache."""

    def __init__(
        self,
        size: int,
        size_swa: int,
        dtype: torch.dtype,
        device: str,
        kvcache: SWAKVPool,
        need_sort: bool,
    ):
    ...
    def available_size(self):
        raise NotImplementedError()

```

Reply:
ok